rebar.config.script

```diff
@@ -1,8 +1,46 @@
+%% -*- erlang -*-
+
+IsWindows = case os:type() of {win32, _} -> true; {_, _} -> false end,
+
+FilterRelxRocksDb = fun(Apps) ->
+                        case IsWindows of
+                            false ->
+                                Apps;
+                            true ->
+                                Apps -- [rocksdb, mnesia_rocksdb]
+                        end
+                    end,
+
+FilterDepsRocksDb = fun(Deps) ->
+                        case IsWindows of
+                            false ->
+                                Deps;
+                            true ->
+                                [D || {K, _} = D <- Deps, K =/= rocksdb, K =/= mnesia_rocksdb]
```

Comment:
Could this be - in the future - be replaced by explicit matching on deps `{_, _}` rather than list comprehension with filter (if any D in Deps did not match `{K, _}` that would be silently skipped).
This is Win only so no changes in existing archs anyway.